lib/matplotlib/table.py

```diff
@@ -42,7 +71,7 @@ def __init__(self, xy, width, height,
                  ):
 
         # Call base
-        Rectangle.__init__(self, xy, width=width, height=height,
+        Rectangle.__init__(self, xy, width=width, height=height, fill=fill,
```

Comment:
I guess that's an intended new feature (respecting `fill` instead of dropping it)?.

Reply:
Yes. I assume that is was accidentally omitted. There's no point in having `fill` in `__init__` if it's silently dropped.